test/addons/rbd-mirror/start

```diff
@@ -224,20 +220,12 @@ def restart_rbd_mirror_daemon(cluster):
 
 def deploy_vrc_sample(cluster):
     print(f"Applying vrc sample in cluster '{cluster}'")
-    kubectl.apply(
-        "--filename=vrc-sample.yaml",
-        "--namespace=rook-ceph",
-        context=cluster,
-    )
+    kubectl.apply("--kustomize=start-data", context=cluster)
```

Comment:
This is not needed, you can delete now the entire function.

Reply:
Done